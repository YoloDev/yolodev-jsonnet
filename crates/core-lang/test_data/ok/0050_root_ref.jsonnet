{
  local c = $,
  local a = c,
  a::5,
  k::'test',
  assert $.a == 5,
  assert a.a == 5,
}
