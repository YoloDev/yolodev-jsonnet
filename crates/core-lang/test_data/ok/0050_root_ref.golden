{
  _: local $#2 = self,
          c#3 = $#2,
          a#4 = c#3;
    if std#1["equals"]($#2["a"], 5.0)
    then null
    else error "Assertion failed: $.a == 5",
  _: local $#2 = self,
          c#3 = $#2,
          a#4 = c#3;
    if std#1["equals"](a#4["a"], 5.0)
    then null
    else error "Assertion failed: a.a == 5"
  ["a"]:
    local $#2 = self,
          c#3 = $#2,
          a#4 = c#3;
    5.0,
  ["k"]:
    local $#2 = self,
          c#3 = $#2,
          a#4 = c#3;
    "test",
}
